<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>L-Connect 3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
          <property name="styleSheet">
           <string notr="true">QWidget#centralWidget {
            background-color: #0c193a;
        }</string>
          </property>
   <layout class="QHBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="sidebar">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
              <property name="styleSheet">
               <string notr="true">QFrame#sidebar {
            background-color: #081536;
            border-right: 1px solid #2A3441;
        }</string>
              </property>
      <layout class="QVBoxLayout" name="sidebarLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QFrame" name="logoBadge">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame#logoBadge {
            background-color: #081536;
            border: none;
            margin: 15px 0px;
            padding: 10px;
        }</string>
                 </property>
         <layout class="QVBoxLayout" name="logoLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="logoInitialLabel">
            <property name="text">
             <string></string>
            </property>
            <property name="pixmap">
             <pixmap>:/icons/resources/logo.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel#logoInitialLabel {
            background-color: transparent;
        }</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="logoLabel">
            <property name="text">
             <string>L-CONNECT 3</string>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel#logoLabel {
            color: #FFFFFF;
            font-size: 16px;
            font-weight: bold;
        }</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="systemInfoBtn">
         <property name="text">
          <string>System Info</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">        QPushButton {
            background-color: transparent;
            border: none;
            color: #B0B0B0;
            padding: 12px 20px;
            text-align: left;
            font-size: 14px;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
            margin: 2px 10px 2px 0px;
        }

        QPushButton:hover {
            background-color: #2A3441;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
        }

        QPushButton:checked {
            background-color: #1d4e83;
            color: #FFFFFF;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
        }</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fanProfileBtn">
         <property name="text">
          <string>Fan Profile</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">        QPushButton {
            background-color: transparent;
            border: none;
            color: #B0B0B0;
            padding: 12px 20px;
            text-align: left;
            font-size: 14px;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
            margin: 2px 10px 2px 0px;
        }

        QPushButton:hover {
            background-color: #2A3441;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
        }

        QPushButton:checked {
            background-color: #1d4e83;
            color: #FFFFFF;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
        }</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lightingBtn">
         <property name="text">
          <string>Lighting</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">        QPushButton {
            background-color: transparent;
            border: none;
            color: #B0B0B0;
            padding: 12px 20px;
            text-align: left;
            font-size: 14px;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
            margin: 2px 10px 2px 0px;
        }

        QPushButton:hover {
            background-color: #2A3441;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
        }

        QPushButton:checked {
            background-color: #1d4e83;
            color: #FFFFFF;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
        }</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="slInfinityBtn">
         <property name="text">
          <string>SL Infinity</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">        QPushButton {
            background-color: transparent;
            border: none;
            color: #B0B0B0;
            padding: 12px 20px;
            text-align: left;
            font-size: 14px;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
            margin: 2px 10px 2px 0px;
        }

        QPushButton:hover {
            background-color: #2A3441;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
        }

        QPushButton:checked {
            background-color: #1d4e83;
            color: #FFFFFF;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
        }</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settingsBtn">
         <property name="text">
          <string>Settings</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">        QPushButton {
            background-color: transparent;
            border: none;
            color: #B0B0B0;
            padding: 12px 20px;
            text-align: left;
            font-size: 14px;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
            margin: 2px 10px 2px 0px;
        }

        QPushButton:hover {
            background-color: #2A3441;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
        }

        QPushButton:checked {
            background-color: #1d4e83;
            color: #FFFFFF;
            border-top-left-radius: 0px;
            border-bottom-left-radius: 0px;
            border-top-right-radius: 20px;
            border-bottom-right-radius: 20px;
        }</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="sidebarSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="versionLabel">
         <property name="text">
          <string>v1.0.0</string>
         </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel#versionLabel {
            color: #B0B0B0;
            font-size: 10px;
            margin-top: 5px;
        }</string>
                 </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="contentStack">
             <property name="styleSheet">
              <string notr="true">QStackedWidget#contentStack {
            background-color: #0c193a;
        }</string>
             </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>